docker-entrypoint.sh

```diff
@@ -3,5 +3,9 @@ set -e
 
 echo "ENTRYPOINT: Starting docker entrypoint…"
 
+bundle install --quiet
```

Comment:
Not sure about this one. I can see the convience for development purposes but having it here in a script that's run in any environment opens doors to potential issues.

In production we go from a prebuilt and stable dependency tree to requiring the container to always check the lock file and _potentially_ reach out to Rubygems to install. Though it should never need to in production we introduce the possibility that it can, and if it can, it can also fail completely or take so long to Bundle it could cause deployment issues.

As the reader you could be forgiven for reading this line and making the assumption that this is where bundle is run, however it actually is the second "just in case" bundle. If you want to make any real changes to how bundle is run, you would have to ignore this and look into the Dockerfile to make a change.

If we are going to add this, I think we should be explict about only running it in dev or test.